packages/suite-desktop/scripts/build.js

```diff
@@ -29,31 +30,41 @@ const mockPlugin = {
     },
 };
 
-console.log('[Electron Build] Starting...');
-const hrstart = process.hrtime();
-build({
-    entryPoints: ['app.ts', 'preload.ts', ...modules].map(f => path.join(electronSource, f)),
-    platform: 'node',
-    bundle: true,
-    target: 'node14.16.0', // Electron 12
-    external: Object.keys({
-        ...pkg.dependencies,
-        ...pkg.devDependencies,
-    }),
-    tsconfig: path.join(electronSource, 'tsconfig.json'),
-    sourcemap: isDev,
-    minify: !isDev,
-    outdir: path.join(__dirname, '..', 'dist'),
-    define: {
-        'process.env.COMMITHASH': JSON.stringify(gitRevision),
-    },
-    plugins: useMocks ? [mockPlugin] : [],
-})
-    .then(() => {
-        const hrend = process.hrtime(hrstart);
-        console.log(
-            '[Electron Build] Finished in %dms',
-            (hrend[1] / 1000000 + hrend[0] * 1000).toFixed(1),
-        );
-    })
-    .catch(() => process.exit(1));
+const run = async () => {
+    // Fetch pubkey
+    const pubkey = await (
+        await fetch('https://trezor.io/security/satoshilabs-2021-signing-key.asc')
+    ).text();
```

Comment:
> Additionally, if I will have some internet shortage during the build process, will I be able to build it correctly? laughing

I think that is out of scope because you download half of the internet via `yarn` anyway :)).

Reply:
I just put the key in the folder. As @tomasklim mentions, there's no need to pull it for each build. 96c870e

Reply:
+1 for baking the key in